plugins/videobasedtracker/BeaconBasedPoseEstimator.cpp

```diff
@@ -212,10 +212,43 @@ namespace vbtracker {
         // do okay without using it, so leaving it out.
         // @todo Make number of iterations into a parameter.
         bool usePreviousGuess = false;
+        int maxIterations = 5;
+        cv::Mat inlierIndices;
         cv::solvePnPRansac(
             objectPoints, imagePoints, m_cameraMatrix, m_distCoeffs, m_rvec,
-            m_tvec, usePreviousGuess, 20, 8.0f,
-            static_cast<int>(objectPoints.size() - m_permittedOutliers));
+            m_tvec, usePreviousGuess, 5, 8.0f,
```

Comment:
Based on your commit messages, I assumed the presence of a literal 5 here was a mis-resolved merge conflict, so when I rebased I used the variable name instead.

Would be good to have the 8.0f "magic number" named as well.